### PR TITLE
created the pickup-dropoff component

### PR DESCRIPTION
**Pull request by moath-dawood**

### CODE REVIEW

File: src/Components/Homepage/index.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import PickupCard from '../PickupCard'
 
 export const HomePage = () => {
   return (
     <div>
```
Comment (baha-ott) [inline src/Components/Homepage/index.js:6]: main elements instead of div

File: src/Components/PickupCard/Dropoff.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
+import * as React from 'react';
```
Comment (baha-ott) [inline src/Components/PickupCard/Dropoff.js]: no need to import everything, just import what you want to use

File: src/Components/PickupCard/Dropoff.js [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+import { useTheme } from '@mui/material/styles';
+import OutlinedInput from '@mui/material/OutlinedInput';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import { Box, Typography, CardMedia, Grid } from '@mui/material';
```
Comment (baha-ott) [inline src/Components/PickupCard/Dropoff.js]: remove things that you are not using, such as Box

File: src/Components/PickupCard/Dropoff.js [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+const MenuProps = {
+    PaperProps: {
+        style: {
+            maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+            width: 250,
+        },
+    },
+};
+
+const names = [
```
Comment (baha-ott) [inline src/Components/PickupCard/Dropoff.js]: cityNames

File: src/Components/PickupCard/Dropoff.js [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+           
+            <FormControl sx={{ m: 1, width:{xs:100, sm:150, md:200}, mt: 3 }}>
+            <Typography>Locations</Typography>
+                <Select
+                    displayEmpty
+                    value={personName}
+                    onChange={handleChange}
+                    input={<OutlinedInput />}
+                    renderValue={(selected) => {
+                        if (selected.length === 0) {
```
Comment (baha-ott) [inline src/Components/PickupCard/Dropoff.js:71]: use ternary operator

**Review by baha-ott — CHANGES_REQUESTED** [review]
divide components more better pls! 
Good job moath, have a nice work !

**Comment by moath-dawood** [comment]
> divide components more better pls! Good job moath, have a nice work !

Will do better in next components, its clustered like this because its a premade component from mui.

**Review by baha-ott — APPROVED** [review]

**Pull request merged**